src/UI/class-settings-page.php

```diff
@@ -892,17 +932,17 @@ public function print_track_post_types_table( array $args ): void {
 	 *
 	 * @since 3.2.0
 	 *
-	 * @return array<string, string> Key-value pairs with post type and their 'track as' value.
+	 * @return array<string> Key-value pairs with post type and their 'track as' value.
 	 */
 	public function get_tracking_values_for_display(): array {
 		$options = $this->parsely->get_options();
 		$types   = array( 'post', 'page' );
 		$result  = array();
 
 		foreach ( $types as $type ) {
-			$array_key = "track_{$type}_types";
-			if ( array_key_exists( $array_key, $options ) ) {
-				foreach ( $options[ $array_key ] as $post_type ) {
```

Comment:
Second line was giving PHPStan error on `$options` so changed the logic a bit to avoid the error without defining ignore comment.